cassovary-core/src/main/scala/com/twitter/cassovary/util/FastUtilUtils.scala

```diff
@@ -61,4 +61,6 @@ object FastUtilUtils {
 
   def newInt2IntOpenHashMap(): mutable.Map[Int, Int] =
```

Comment:
I cannot answer, but you are right that wrapping the fastutils collection is expensive. See #142